src/delay.rs

```diff
@@ -92,21 +92,22 @@ impl From<TickType> for Option<Duration> {
 /// trigger.
 pub struct Ets;
 
-// No longer available in the generated bindings for ESP-IDF 5
+// Not available in the generated bindings for ESP-IDF 4
 #[cfg(not(esp_idf_version_major = "4"))]
 extern "C" {
     pub fn ets_delay_us(us: u32);
 }
 
 impl Ets {
-    #[deprecated = "Use delay_ns instead"]
     pub fn delay_us(us: u32) {
```

Comment:
This method was deprecated. You have un-deprecated it. Any reason for that? Perhaps you do not understand, that `delay_us` and `delay_ns` **are the same thing** and we have deprecated the old `delay_us` name because we have the new `delay_ns` name? (We follow here `embedded-hal` naming)

Reply:
while i agree overall, embedded-hal did not remove delay_us. What they did is rename the overall delay_provider to DelayNs and implement all 3 methods delay_ns/us/ms for that. So we may should not "deprecate" the delay_us method itself from the public api

Reply:
https://github.com/rust-embedded/embedded-hal/blob/master/embedded-hal/src/delay.rs#L9

Reply:
I agree. Also... I'm an idiot - us stands for *microseconds*, so it is not the same as *nanoseconds*.
Let me take a second look...

Reply:
Does code support utf8 ? :D we could use delay_µs and hope everybody has good autocomplete  :smile: